mingw-w64-crystal/PKGBUILD

```
@@ -31,16 +32,19 @@ depends=(
 )
 makedepends=(
   "git"
+  $( (( !_bootstrap )) && echo "${MINGW_PACKAGE_PREFIX}-crystal")
```

nit: we usually do something like
```suggestion
  $( (( _bootstrap )) || echo "${MINGW_PACKAGE_PREFIX}-crystal")
```